css/selectors/old-tests/WEB_FEATURES.yml

```diff
@@ -3,3 +3,9 @@ features:
   files:
   - "css3-modsel-21*"
   - "css3-modsel-66*"
+- name: root
+  files:
+  - css3-modsel-27.xml
+  - css3-modsel-27a.xml
```

Comment:
Excluded [`css3-modsel-27b.xml`](https://github.com/bocoup/wpt/blob/web-features-root-selector/css/selectors/old-tests/css3-modsel-27b.xml) since it tests * html and * :root simultaneously.